object_detection/scripts/get_objects_and_coordinates.py

```diff
@@ -24,31 +25,30 @@
 """
 
 import os
+# Run tensorflow CPU instead of GPU (because the Jetson Nano runs out of memory when using GPU)
+os.environ["CUDA_VISIBLE_DEVICES"]="-1" 
+import sys
+import json
+
 import argparse
 import cv2
 import numpy as np
 import tensorflow as tf
-import sys
-import json
 
 from utils import label_map_util
 from utils import visualization_utils as vis_util
 
-# this will change when testing on the Jetson
-from google.colab.patches import cv2_imshow
-
 def run_model(image_directory):
     """
     This function loads the neccesary files, runs the model and gets the detection boxes, scores and classes.
     """
     global boxes, scores, classes, num, category_index, image
 
     MODEL_NAME = 'inference_graph'
-    IMAGE_NAME = image_directory
     CWD_PATH = os.getcwd()
     PATH_TO_CKPT = os.path.join(CWD_PATH,MODEL_NAME,'frozen_inference_graph.pb')
     PATH_TO_LABELS = os.path.join(CWD_PATH,'training','labelmap.pbtxt')
-    PATH_TO_IMAGE = os.path.join(CWD_PATH,IMAGE_NAME)
+    PATH_TO_IMAGE = image_directory
```

Comment:
So now the absolute path must be given ?

Reply:
Well, it is not relative only anymore. Now it can be absolute or relative path.